### PR TITLE
WRN-17309: [DrawingComponent]Add 'disabled' functionality

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Added disabled styles

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-17309

### CODE REVIEW

File: Drawing/Drawing.module.less [hunk 1]
```diff
@@ -4,8 +4,8 @@
 
 .drawing {
 	.applySkins({
-		.disabled({
-			opacity: 0.5;
-		})
+		.disabled{
```
Comment (stanca-pop-lgp) [inline Drawing/Drawing.module.less:7]: Please add an empty space before {

File: samples/sampler/stories/default/Drawing.js [hunk 2]
```diff
@@ -8,11 +8,23 @@ export default {
...
+				<p style={{fontSize: '70%', fontStyle: 'italic'}}>
+					<sup>*</sup>Drawing is not allowed while <code>disabled</code> is true.
+				</p>
+			) : (
+				<p />
+			)}
+			<Drawing
+				disabled={disabled}
+			/>
+		</section>
```
Comment (stanca-pop-lgp) [inline samples/sampler/stories/default/Drawing.js:25]:
```suggestion
		<section>
			<p style={{visibility: disabled ? 'visible' : 'hidden', fontSize: '70%', fontStyle: 'italic'}}>
				<sup>*</sup>Drawing is not allowed while <code>disabled</code> is true.
			</p>
			<Drawing
				disabled={disabled}
			/>
		</section>
```

Comment (stanca-pop-lgp) [reply samples/sampler/stories/default/Drawing.js:25]: Adding an empty paragraph doesn't take the space for the text that appears when disabled is true. You can work with visibility.

**Review by stanca-pop-lgp — APPROVED** [review]
LGTM

**Review by daniel-stoian-lgp — APPROVED** [review]

**Pull request merged**